top-tab/style.css

```diff
@@ -0,0 +1,14 @@
+body {
+    background-image: url("https://source.unsplash.com/3840x2160/?nature");
```

Comment:
I ended up loading my background with javascript (thinking i could add a button to reload it when the user wanted to) but this is much cleaner.

Reply:
Thanks!